packages/backend/src/database/entities/organizationMemberships.ts

```diff
@@ -1,25 +1,21 @@
+import { OrganizationMemberRole } from 'common';
 import { Knex } from 'knex';
 
+export const OrganizationMembershipsTableName = 'organization_memberships';
+
 export type DbOrganizationMembership = {
     organization_id: number;
     user_id: number;
     created_at: Date;
+    role: OrganizationMemberRole;
 };
 
 export type DbOrganizationMembershipIn = {
     user_id: number;
     organization_id: number;
 };
```

Comment:
@owlas We should require 'role' when we add a membership, otherwise all the invited users will be admins by default.